controller/system_rollout_controller.go

```diff
@@ -951,6 +951,11 @@ func (c *SystemRolloutController) restoreConfigMaps() (err error) {
 	for _, restore := range c.configMapList.Items {
 		log := c.logger.WithField(types.KubernetesKindConfigMap, restore.Name)
 
+		if restore.Name == types.DefaultDefaultSettingConfigMapName {
+			log.Infof(SystemRolloutMsgIgnoreItemFmt, types.DefaultDefaultSettingConfigMapName)
+			continue
+		}
```

Comment:
So basically, the default config will not be taken care during system restored? because we will just rely on setting restoring which will exclude some settings as below potentially impact the restore process, right?

If yes, we probably need to describe this behavior also mentioned what settings will not be restored in the doc. https://longhorn.io/docs/1.4.0/advanced-resources/system-backup-restore/restore-longhorn-system/#prerequisite

```go
var systemRolloutIgnoredSettings = [...]string{
	string(types.SettingNameConcurrentBackupRestorePerNodeLimit),
	string(types.SettingNameConcurrentReplicaRebuildPerNodeLimit),
	string(types.SettingNameBackupTarget),
	string(types.SettingNameBackupTargetCredentialSecret),
}

```